### PR TITLE
start vm after shutdown and execute command on it

### PR DESCRIPTION
**Pull request by ahmedelsayed-93**
fixes https://github.com/zero-os/0-orchestrator/issues/1169

### CODE REVIEW

**Comment by ahmedelsayed-93** [comment]
```
 % nosetests -s -v testcases/basic_tests/test07_vms_apis.py:TestVmsAPI --tc-file config.ini
GAT-067 ... ok
GAT-068 ... ok
GAT-070 ... ok
GAT-071 ... ok
GAT-072 ... ok
GAT-073 ... ok
GAT-075 ... ok
GAT-076 ... SKIP: https://github.com/zero-os/0-orchestrator/issues/994
GAT-077 [with *args=('same_node',)] ... SKIP: https://github.com/zero-os/0-orchestrator/issues/1199
GAT-077 [with *args=('different_node',)] ... SKIP: https://github.com/zero-os/0-orchestrator/issues/1199
GAT-077 ... ok

----------------------------------------------------------------------
Ran 11 tests in 1662.148s

OK (SKIP=3)
```

File: tests/0_orchestrator/test_suite/testcases/basic_tests/test07_vms_apis.py [hunk 1]
```diff
@@ -380,10 +374,24 @@ def test010_post_nodes_vms_vmid_shutdown(self):
         core0_list_vms = self.core0_client.client.kvm.list()
         self.assertNotIn(self.data['id'], [x['name'] for x in core0_list_vms])
 
-        elf.lg.info(' [*] Execute command on virtual machine (VM0), should fail')
+        elf.lg.info('[*] Execute command on virtual machine (VM0), should fail')
```
Comment (islamTaha12) [inline tests/0_orchestrator/test_suite/testcases/basic_tests/test07_vms_apis.py]: self not elf

File: tests/0_orchestrator/test_suite/testcases/basic_tests/test07_vms_apis.py [hunk 2]
```diff
@@ -380,10 +374,24 @@ def test010_post_nodes_vms_vmid_shutdown(self):
...
         response = self.execute_command_inside_vm(self.ssh_client, self.vm_ip_address, 'uname')
         self.assertEqual(response.state, 'ERROR')
 
+        self.lg.info('[*] Start virtual machine (VM0), should succeed with 204')
+        response = self.vms_api.post_nodes_vms_vmid_start(self.nodeid, self.data['id'])
```
Comment (islamTaha12) [inline tests/0_orchestrator/test_suite/testcases/basic_tests/test07_vms_apis.py]: Maybe u have to wait for awhile tell the machine to be booting

Comment (ahmedelsayed-93) [reply tests/0_orchestrator/test_suite/testcases/basic_tests/test07_vms_apis.py]: shutdown and start vm apis wait until the action done

Comment (ahmedelsayed-93) [reply tests/0_orchestrator/test_suite/testcases/basic_tests/test07_vms_apis.py]: all post apis like start, stop, shutdown, pause, and resume

**Pull request merged**